VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -406,12 +406,27 @@ private static void InitializePlatform(IAppBuilder app, IUnityContainer containe
             app.SanitizeThreadCulture();
             ICacheManager<object> cacheManager = null;
 
+            // Try to find app setting which controls using of scaled up VC cache instances
+            var cacheScaleUpSetting = ConfigurationManager.AppSettings["VirtoCommerce:Cache:ScaleUpEnabled"];
```

Comment:
I think we should replace of using the special flag to read the redisConnection from web.config   <connectionStrings> section.  that can be taken by follow call in startup.cs.
CacheManager 
```C#
ConfigurationHelper.GetConnectionStringValue("redisConnectionString")
```
 Need somehow find out solution how to use this connection string instead of special configuration section. Because it is impossible to manage this custom section in Azure portal
```XML
<cacheManager.Redis>
        <connections>
...
```